lib/src/kbasesearchengine/main/IndexerWorker.java

```diff
@@ -550,14 +550,13 @@ private String toVerRep(final SearchObjectType globalObjectType) {
     private void indexObjectInStorage(
             final GUID guid,
             final Instant timestamp,
-            final boolean isPublic,
             final SourceData obj,
             final ObjectTypeParsingRules rule,
             final Map<GUID, ParsedObject> guidToObj,
             final String parentJson)
             throws InterruptedException, IndexingException {
         final List<?> input = Arrays.asList(rule, obj, timestamp, parentJson, guid, guidToObj,
-                isPublic);
+                obj.isPublic());
```

Comment:
isPublic is contained in obj, so why pass both? Follow the path of obj though the code until is public is needed separate from obj and *then* split them up.

Reply:
Done